api/server/call_logs.go

```diff
@@ -12,35 +14,25 @@ func (s *Server) handleCallLogGet(c *gin.Context) {
 
 	appName := c.MustGet(api.AppName).(string)
 	callID := c.Param(api.Call)
-	_, err := s.Datastore.GetCall(ctx, appName, callID)
-	if err != nil {
-		handleErrorResponse(c, err)
-		return
-	}
 
-	callObj, err := s.LogDB.GetLog(ctx, appName, callID)
+	logReader, err := s.LogDB.GetLog(ctx, appName, callID)
 	if err != nil {
 		handleErrorResponse(c, err)
 		return
 	}
 
-	c.JSON(http.StatusOK, callLogResponse{"Successfully loaded call", callObj})
-}
-
-func (s *Server) handleCallLogDelete(c *gin.Context) {
-	ctx := c.Request.Context()
-
-	appName := c.MustGet(api.AppName).(string)
-	callID := c.Param(api.Call)
-	_, err := s.Datastore.GetCall(ctx, appName, callID)
-	if err != nil {
-		handleErrorResponse(c, err)
-		return
+	// TODO this API needs to change to text/plain / gzip anyway, punting
```

Comment:
Isn't this it the right time to do so?

Reply:
as stated in parent, I have another WIP patch working on this. but having s3 api is blocking for the service piece and this isn't really critical.